outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
```

Comment:
I'm into it. It sounds like something to look at in concert with rationalizing / cataloging the docs as we've discussed before. Did you make progress on that yet?